ding/framework/context.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import dataclasses
+import torch
```

Comment:
why import torch here

Reply:
Because in line 56 I used torch.Tensor for typing

Reply:
use `TYPE_CHECKING` flag like [this](https://github.com/opendilab/DI-engine/blob/main/ding/framework/middleware/functional/ctx_helper.py#L6)